apps/nextjs/src/components/manage/boards/import-board-modal.tsx

```diff
@@ -0,0 +1,191 @@
+            console.error(result.error.errors);
+            setFileValid(false);
+            return;
+          }
+
+          setFileValid(true);
+          form.setFieldValue("configuration.name", result.data.configProperties.name);
+        })();
+      },
+    },
+  );
+
+  const { mutateAsync } = clientApi.board.importOldmarrConfig.useMutation();
+
+  const handleSubmitAsync = async (values: { file: File; configuration: OldmarrImportConfiguration }) => {
+    const formData = new FormData();
+    formData.set("file", values.file);
+    formData.set("configuration", JSON.stringify(values.configuration));
+
+    await mutateAsync(formData, {
+      onSuccess() {
+        actions.closeModal();
+      },
+    });
+  };
+
+  return (
+    <form
+      onSubmit={form.onSubmit((values) => {
+        if (!fileValid) {
+          return;
+        }
+
+        void handleSubmitAsync({
+          // It's checked for null in the superrefine
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          file: values.file!,
+          configuration: values.configuration,
+        });
+      })}
+    >
+      <Stack>
+        <FileInput
+          accept="application/json"
+          {...form.getInputProps("file")}
+          error={
+            (form.getInputProps("file").error as string | undefined) ??
+            (!fileValid ? "Invalid configuration file" : undefined)
+          }
+          type="button"
+          label="Select JSON file"
+        />
+
+        <Fieldset legend="Apps">
+          <Grid>
+            <Grid.Col span={{ base: 12, sm: 6 }}>
+              <Switch
+                label="Avoid duplicates"
+                {...form.getInputProps("configuration.distinctAppsByHref", { type: "checkbox" })}
+              />
+            </Grid.Col>
+            <Grid.Col span={{ base: 12, sm: 6 }}>
+              <Switch
+                label="Only import apps"
+                {...form.getInputProps("configuration.onlyImportApps", { type: "checkbox" })}
+              />
+            </Grid.Col>
+          </Grid>
+        </Fieldset>
+
+        <TextInput label="Board name" {...form.getInputProps("configuration.name")} />
+
+        <Radio.Group label="Screen size" {...form.getInputProps("configuration.screenSize")}>
+          <Group mt="xs">
+            <Radio value="sm" label="Small" />
+            <Radio value="md" label="Medium" />
+            <Radio value="lg" label="Large" />
+          </Group>
+        </Radio.Group>
+
+        <SelectWithDescription
+          label="Sidebar behaviour"
+          description="Sidebars were removed in 1.0, you can select what should happen with the items inside them."
+          data={[
+            {
+              value: "last-section",
+              label: "Last section",
+              description: "Sidebar will be displayed below the last section",
+            },
+            {
+              value: "remove-items",
+              label: "Remove items",
```

Comment:
Test if this works

Reply:
Okay sidebars do not quite work yet, I'll need to test it further and fix the issues

Reply:
Okay just tested it again and it actually works as the sidebars were only moved in the md sized gridstack. So I imported md and everything worked as expected